03-Security-In-CICD/Readme.md

```diff
@@ -125,9 +125,10 @@ while running the AzSK SVTs manually - you can choose to specify the target reso
 a {tagname, tagvalue} pair based on how your application's resources are organized.  
 When the pipeline executes, SVTs will scan the specified set of resources.
 
-Along with input parameter, you can check for below options
-<br/>**Enable LAWS Logging:** Switch to enable this task to publish SVT evaluation results to a Log Analytics workspace. Steps to configure workspace credentials are explained in Step-4
-<br/>**Aggregate Control Status:** Switch to aggregate the SVTs control output. When this is turned off it would show all the failed individual controls in the task summary output.
+Apart from input parameter, you can also use checkboxes to enable/disable one or more below options
+<br/>**Also scan subscription controls:** Switch to scan subscription security controls. This will require Azure Connection (SPN) has reader access of the entire subscription.
```

Comment:
This will require Azure Connection (SPN) to have reader access at the subscription scope.